models/dpp.go

```diff
@@ -11,67 +11,29 @@ type DPPPayRequest struct {
-	SPVRequired bool         `json:"spvRequired"`
-	Network     string       `json:"network"`
-	Outputs     []Output     `json:"outputs"`
-	Fees        *bt.FeeQuote `json:"fees"`
-	CreatedAt   time.Time    `json:"createdAt"`
-	ExpiresAt   time.Time    `json:"expiresAt"`
-}
-
-// PaymentRequest message used in BIP270.
-// See https://github.com/moneybutton/bips/blob/master/bip-0270.mediawiki#paymentrequest
+// PaymentRequestResponse
 type PaymentRequest struct {
-	// Network  Always set to "bitcoin" (but seems to be set to 'bitcoin-sv'
-	// outside bip270 spec, see https://handcash.github.io/handcash-merchant-integration/#/merchant-payments)
-	// {enum: bitcoin, bitcoin-sv, test}
-	// Required.
-	Network string `json:"network" example:"mainnet" enums:"mainnet,testnet,stn,regtest"`
-	// SPVRequired if true will expect the sender to submit an SPVEnvelope in the payment request, otherwise
-	// a rawTx will be required.
-	SPVRequired bool `json:"spvRequired" example:"true"`
-	// Destinations contains supported payment destinations by the merchant and dpp server, initial P2PKH outputs but can be extended.
-	// Required.
-	Destinations PaymentDestinations `json:"destinations"`
-	// CreationTimestamp Unix timestamp (seconds since 1-Jan-1970 UTC) when the PaymentRequest was created.
-	// Required.
-	CreationTimestamp time.Time `json:"creationTimestamp" swaggertype:"primitive,string" example:"2019-10-12T07:20:50.52Z"`
-	// ExpirationTimestamp Unix timestamp (UTC) after which the PaymentRequest should be considered invalid.
-	// Optional.
-	ExpirationTimestamp time.Time `json:"expirationTimestamp" swaggertype:"primitive,string" example:"2019-10-12T07:20:50.52Z"`
-	// PaymentURL secure HTTPS location where a Payment message (see below) will be sent to obtain a PaymentACK.
-	// Maximum length is 4000 characters
-	PaymentURL string `json:"paymentUrl" example:"https://localhost:3443/api/v1/payment/123456"`
-	// Memo Optional note that should be displayed to the customer, explaining what this PaymentRequest is for.
-	// Maximum length is 50 characters.
-	Memo string `json:"memo" example:"invoice number 123456"`
-	// MerchantData contains arbitrary data that may be used by the payment host to identify the PaymentRequest.
-	// May be omitted if the payment host does not need to associate Payments with PaymentRequest
-	// or if they associate each PaymentRequest with a separate payment address.
-	// Maximum length is 10000 characters.
-	MerchantData *Merchant `json:"merchantData,omitempty"`
-	// FeeRate defines the amount of fees a users wallet should add to the payment
-	// when submitting their final payments.
-	FeeRate *bt.FeeQuote `json:"fees"`
+	Network             string         `json:"network"`
+	Destinations        DPPDestination `json:"destinations"`
+	CreationTimestamp   time.Time      `json:"creationTimestamp"`
+	ExpirationTimestamp time.Time      `json:"expirationTimestamp"`
+	PaymentURL          string         `json:"paymentURL"`
```

Comment:
shouldn't this be just `url` in the json?

Reply:
no
<img width="380" alt="image" src="https://user-images.githubusercontent.com/8416253/164760752-a430d73a-b1bb-4a7c-b6e4-65672fb6f58e.png">